projects/demo/src/modules/app/pages.ts

```diff
@@ -267,6 +267,12 @@ export const pages: DocRoutePages = [
                     name: 'tui-button-group-card',
                 },
             },
+            {
+                section: 'Components',
+                title: 'ButtonSelect',
+                keywords: 'кнопка, button, select, multiselect',
+                route: DemoRoute.ButtonSelect,
```

Comment:
Add it to this "See also" group too.

https://github.com/taiga-family/taiga-ui/blob/d7a78e370ac999a6019a1294a88c230af6ca2d43/projects/demo/src/modules/app/app.const.ts#L21

So, user will have two opportunities to find new component:
* `Buttons` sidebar navigation group
* `See also` group for select-like entities